dt-core/admin/menu/tabs/tab-customizations.php

```diff
@@ -82,7 +82,16 @@ public function admin_enqueue_scripts() {
         wp_enqueue_style( 'dt_settings_css' );
 
         $post_type = self::get_parameter( 'post_type' );
-        if ( !isset( $post_type ) || is_null( $post_type ) ) {
+        if ( !isset( $post_type ) || is_null( $post_type ) ){
```

Comment:
@kodinkat is this section needed since the dt-settings bellow has these options?

Reply:
Yes, otherwise an exception is thrown when attempting to create a new record type prior to selecting any existing ones.

![Screenshot 2023-07-06 at 12 08 24](https://github.com/DiscipleTools/disciple-tools-theme/assets/19330800/f6450231-5727-4384-a2b8-45042f9aabb7)

Reply:
I moved the code around a little to easily add to the base object.